### DISCUSSION TITLE
window position on macOS

### DISCUSSION THREAD

**Discussion by applebait** [kovidgoyal/kitty#5048] [Q&A] [answered]
Alongside with kitty I'm using Amethyst. Amethyst is a Tile Window Manager (TWM), also and obviously, available for macOS. Every other Terminal Emulator I have installed behaves as it should, I only use this TWM with Terminal Emulators but kitty is giving me a head burn because I cannot make it to have the same distance for the screen edges on all its sides. Somehow kitty is forcing a value that is overtaking Amethyst, both Alacritty, iTerm2 and Terminal follow Amethyst.
The only way I can get it with the same margins to the screen edges is by using another utility, MultiTouch or Swish, to center any window in the screen.
So, what am I missing here? Is there any line in the config I'm missing? Yes, I know, I admit it, I'm picky! :)

Thanks in advance everyone.

**Comment by kovidgoyal** [comment, marked as answer]
kitty does not control its window position, only window size. That is up to the window
manager. Why this particular one doesnt like kitty I have no idea, you
would need to ask its developers.

**Reply by applebait** [reply]
I will, thank you! :)
